helmfile/charts/prometheus-alerts/templates/alerts/bucket-daily-checks.yaml

```diff
@@ -17,57 +17,12 @@ spec:
   groups:
   - name: bucket-daily-checks
     rules:
-    - alert: HarborBucket36hActivityCheck
+    - alert: Bucket36hActivityCheck
       annotations:
-        description: The bucket {{.Values.buckets.harbor}} havent had any activity in 36h hours.
-        summary: The bucket {{.Values.buckets.harbor}} havent had any activity in 36h hours.
+        message: The bucket {{`{{ $labels.bucket }}`}} havent had any activity in 36h hours.
       expr: |-
         (
-          (time() - s3_last_modified_object_date{bucket={{.Values.buckets.harbor | quote}}}) / 3600 > 36
```

Comment:
Could we merge this file with `daily-checks.yaml`? It seems like both just have s3 alerts.